articles/openshift/oshift-how-use-portworx-shared-volumes.md

```diff
@@ -0,0 +1,238 @@
+                - matchExpressions:
+                  - key: node-role.kubernetes.io/compute
+                    operator: Exists
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - weight: 100
+                podAffinityTerm:
+                  labelSelector:
+                    matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                      - portworx-sharedvolumedemo
+                  topologyKey: kubernetes.io/hostname
+          volumes:
+          - name: shared-data
+            persistentVolumeClaim:
+              claimName: pvc-shared
+          containers:
+          - name: debian-container
+            image: debian
+            volumeMounts:
+            - name: shared-data
+              mountPath: /pod-data
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: "250m"
+              requests:
+                cpu: "250m"
+            command: ["/bin/sh"]
+            args:
+              - "-c"
+              - >
+                while true; do
+                  sleep 300;
+                done
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: portworx-sharedvolumedemo-b
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: portworx-sharedvolumedemo
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: node-role.kubernetes.io/compute
+                    operator: Exists
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - weight: 100
+                podAffinityTerm:
+                  labelSelector:
+                    matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                      - portworx-sharedvolumedemo
+                  topologyKey: kubernetes.io/hostname
+          volumes:
+          - name: shared-data
+            persistentVolumeClaim:
+              claimName: pvc-shared
+          containers:
+          - name: debian-container
+            image: debian
+            volumeMounts:
+            - name: shared-data
+              mountPath: /pod-data
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: "250m"
+              requests:
+                cpu: "250m"
+            command: ["/bin/sh"]
+            args:
+              - "-c"
+              - >
+                while true; do
+                  date >> /pod-data/sharedfile;
+                  echo Hello from the second pod >> /pod-data/sharedfile;
+                  sleep 1;
+                done
+    ```
+
```

Comment:
It'd be useful here to call out some of the details of the above - for example where we set anti-affinity and maybe a link to where more can be read about that.

Reply:
Is what I've added in commit c9d4284 what you had in mind?